source/features/cross-deleted-pr-branches.tsx

```diff
@@ -8,29 +8,29 @@ import features from '.';
 
 function init(): void | false {
 	const lastBranchAction = select.last('.TimelineItem-body .user-select-contain > span:not(.base-ref)');
```

Comment:
Does this work?
```suggestion
	const lastBranchAction = select.last('.TimelineItem-body .user-select-contain > .head-ref');
```

Reply:
No it will fail here https://github.com/sindresorhus/refined-github/pull/271

Since the deleted branch does not have `.head-ref` after its deleted

Reply:
Indeed, how about this?
```suggestion
	const lastBranchAction = select.last('.TimelineItem-body .commit-ref');
```

Reply:
No that was not good. We only want if they deleted, that gives any action with a commit ref.

Reply:
If a branch has been deleted, there are no other commit refs afterwards, I think

Reply:
No but that selector will catch the merged into action

Reply:
But I see that .`user-select-contain` here is actually used to find the branch modified event. So
```suggestion
	const lastBranchAction = select.last('.TimelineItem-body .user-select-contain.commit-ref');
```

Reply:
Looks like the old code was to exclude exactly this https://github.com/jquery/jquery/pull/769.

Reply:
> In [jquery/jquery#769](https://github.com/jquery/jquery/pull/769) it selects the base branch but then it's filtered out by the next check (`!includes(' deleted ')`)

Great!

Reply:
> In [jquery/jquery#769](https://github.com/jquery/jquery/pull/769) it selects the base branch but then it's filtered out by the next check (`!includes(' deleted ')`)

Just realized that will cause a return false.

If we add :not(.base-ref) it will exclude it.

Reply:
> Just realized that will cause a return false.

And? If the last event is not a deletion, then the branch hasn't been deleted. That PR is locked, so even if it's deleted, we can't detect that.

Reply:
Everything is working fine, nothing is broken. The only thing is the feature logger.

As we did the following.
```js
	if (!headReferenceLink && !lastBranchAction) {
		return; // Don't return false, This feature’s CSS already takes care of this
	}
```

Reply:
@fregante if this is a non-issue, lets merge this.